www/src/templates/template-remote-packages.js

```diff
@@ -0,0 +1,99 @@
+import React from "react"
+import Helmet from "react-helmet"
+import distanceInWords from 'date-fns/distance_in_words'
+import typography, { rhythm } from "../utils/typography"
+
+import MarkdownPageFooter from "../components/markdown-page-footer"
+
+import Container from "../components/container"
+
+class RemotePackagesTemplate extends React.Component {
+  render(){
+    const deprecated = this.props.data.npmPackage.deprecated === "false" ? false : true
+    const page = this.props.data.npmPackage.readme.childMarkdownRemark
+    const info = this.props.data.npmPackage
+    const lastUpdated = `${distanceInWords(new Date(info.modified), new Date())} ago`;
```

Comment:
this is confusing, `info` should be named `npmPackage`, and having the other variables is also confusing, just use eg `npmPackage.deprecated` in the component hierarchy below